src/test/java/api/requests/scenarios/LoanDueDatesAfterRecallTests.java

```diff
@@ -862,12 +862,21 @@ void shouldExtendLoanDueDateByAlternatePeriodWhenOverdueLoanIsRecalledAndPolicyA
 
     final JsonObject storedLoan = loansStorageClient.getById(loan.getId()).getJson();
 
-    assertThat(storedLoan.getString("dueDate"), withinSecondsBefore(30, expectedLoanDueDate));
+    assertThat(storedLoan.getString(DUE_DATE), withinSecondsBefore(30, expectedLoanDueDate));
     // Verify published loan event type description
-    var publishedLoanEvents = FakePubSub.getPublishedEventsAsList(byLogEventType(LOAN));
+    var publishedLoanLogEvents = FakePubSub.getPublishedEventsAsList(byLogEventType(LOAN));
+
+    assertThat(publishedLoanLogEvents.size(), is(2));
+    verifyPublishedLogEventDescription(publishedLoanLogEvents.get(0), loan, expectedLoanDueDate);
+    verifyPublishedLogEventDescription(publishedLoanLogEvents.get(1), loan, expectedLoanDueDate);
+  }
+
+  private void verifyPublishedLogEventDescription(JsonObject eventLogJsonObject,
```

Comment:
Again, `verifyPublishedLogEventDescription` name is very generic, we get nothing from it other than it's related to log events, but it does a very specific thing for one particular kind of events. Let's rename it to something like verifyLogEventDueDateChangedMessage

Reply:
done